common/persistence/nosql/nosqlplugin/cassandra/workflowUtils.go

```diff
@@ -1032,27 +1056,27 @@ func (db *cdb) createWorkflowExecutionWithMergeMaps(
 		return fmt.Errorf("should only support WorkflowExecutionMapsWriteModeCreate")
 	}
 
-	err = db.updateActivityInfos(batch, shardID, domainID, workflowID, execution.RunID, execution.ActivityInfos, nil)
+	err = db.updateActivityInfos(batch, shardID, domainID, workflowID, execution.RunID, execution.ActivityInfos, nil, 0)
```

Comment:
If activities are created in workflow creation without ttl and never updated, these activities won't have ttls attached.

Reply:
but this is the create workflow execution flow. the execution in this flow has not ttl to begin with. :'( Also in case of `delay start` etc we can not have ttls attached to workflows that have not been started which translates to activities as well.

Should I just add an exception note here?